config/default/manager_credentials_patch.yaml

```diff
@@ -12,11 +12,11 @@ spec:
         - name: VSPHERE_USERNAME
           valueFrom:
             secretKeyRef:
-              name: capv-manager-bootstrap-credentials
+              name: manager-bootstrap-credentials
               key: username
         - name: VSPHERE_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: capv-manager-bootstrap-credentials
```

Comment:
@randomvariable - can we document this change on the release note ?

Reply:
Don't think this is a change rather than a fixup. Because of the nested kustomize runs, this was coming out as `capv-capv-manager-bootstrap-credentials`, kustomize now correctly prepends `capv-` once